runtime/frequency/src/lib.rs

```diff
@@ -330,7 +330,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("frequency"),
 	impl_name: create_runtime_str!("frequency"),
 	authoring_version: 1,
-	spec_version: 75,
+	spec_version: 77,
```

Comment:
Just checking, is this expecting another PR to go in first?

Reply:
I might need to change it to 78. Since there are 2 other PR's that can go it before this to deconflict.